articles/architecture-scenarios/implementation/b2c/planning.md

```diff
@@ -0,0 +1,128 @@
+:::
+
+The evidence we've collected suggests that customers who also adopt an iterative release process make better progress. For example, let's say that you have 3-4 applications you want to integrate with Auth0. Rather than integrating all at once and shipping only when that's done, you might consider completing the integration for one application before moving on to the next. This way, your teams can benefit from the experience of working with Auth0 and leverage this experience to help increase the velocity of each iteration.
+
+::: warning
+While adopting an iterative release process will improve your time to market when integrating multiple applications, there are additional considerations for you to review if you want to support [Single Sign-On (SSO)](/sso/current).
+:::
+
+In addition, there may be other groups within your organization who have worked with Auth0 and from whom you can go to for first-hand experience - it's not uncommon for us to work with different parts of the same company at different points in time. We may be able to help you identify these groups prior to the beginning of your project, which might help you as you structure your initial project plans.
+
+## Phase 1
+
+Phase 1 focuses on integrating your application(s) with Auth0. During phase 1 you’ll tackle the 10 key stages to Go-Live, across 3 key milestones, and, by tackling the main risk items, will address the most critical paths associated with integration. At the end of this phase you’ll have working implementation that can be taken into production - or at the very least be provided as part of an early adopter or Beta program. 
+
+Phase 1 consists of a number of workstreams, with a number of topics in each. The workstreams, topics and the order in which you address each is important, so we recommend you follow the guidance prescribed. That’s not to say you can’t or shouldn’t tackle work in parallel: User Provisioning and User Authentication, for example, could be tackled independently and at the same time, and these could both be tackled in parallel with your Branding efforts. In the majority of successful integration cases we’ve also found that different teams tackle different streams, and that this can provide significant benefit: your design and development team(s) would typically tackle implementation whilst at the same time your branding team would tackle Auth0 asset customization thus reducing overall time to market.
+
+The diagram below provides an overall view of the planning associated with Phase 1. By Milestone 1, you will have completed major work required to integrate an application, will have addressed the most significant risk items, and will also be able to provide demonstrable functionality to stakeholders. 
+
+<img src="/media/articles/architecture-scenarios/b2c-project-plan-milestones.png" alt="B2C Project Plan Milestones" data-zoomable>
```

Comment:
I'm made some final adjustments to the diagram to improve clarity. If we can incorporate those changes that would be great 👍 (https://docs.google.com/document/d/1eID42s9-HZsOvXBdHI4emxuJm10Y2vJk7jM4bMM433s/edit?usp=sharing). Thank you.